tests/functional/test_config_groups.py

```diff
@@ -448,6 +418,24 @@ def _get_config_from_group(self, cluster_group: GroupConfig):
         config_group["file"] = config_group["file"].replace("\n", "")
         return config_group
 
+    @allure.step('Check error that group config cant change')
+    def _check_error_with_adding_param_to_group(self, group: GroupConfig, params: dict):
+        with allure.step(f'Check that error is "{GROUP_CONFIG_HOST_EXISTS.code}"'):
+            with pytest.raises(ErrorMessage) as e:
+                group.config_set_diff(params)
+            ATTRIBUTE_ERROR.equal(e)
+        with allure.step(f'Check error message is "{HOST_EXISTS_MESSAGE}"'):
+            assert (
+                ATTRIBUTE_ERROR_MESSAGE in e.value.error['desc']
+            ), f"Should be error message '{ATTRIBUTE_ERROR_MESSAGE}'"
+
+    def _check_error_in_all_params_in_group(self, group: GroupConfig, config_before: dict):
+        for param in config_before.keys():
```

Comment:
add allure step for each param
now it is not obviously
![image](https://user-images.githubusercontent.com/31664571/136325871-6139609e-b0f3-497b-b871-dc6401d993d9.png)
upd: will be fixed in #1144